app/actions/views/account_notifications.js

```diff
@@ -21,9 +21,27 @@ export function handleUpdateUserNotifyProps(notifyProps) {
                 value: interval
             }];
 
-            await savePreferences(emailInterval)(dispatch, getState);
+            dispatch({
```

Comment:
I thought the idea was to leave the original actions intact and just use optimistic action types?